contact.html

```diff
@@ -1,41 +1,41 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <link rel="stylesheet" href="contact.css" />
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>contact</title>
-  </head>
-  <body>
-    <div>
-      <nav>
-        <ul>
-          <li><a href="index.html">Home</a></li>
-          <li><a href="aboutme.html">About Me</a></li>
-          <li><a href="project.html">Projects</a></li>
-          <li style="float: right"><a href="contact.html">Contact</a></li>
-        </ul>
-      </nav>
-    </div>
-    <div class="containor">
-      <div class="heading">
-        <h1>Contact</h1>
-      </div>
-
-      <div class="contact">
-        <p>full name: chidera Njoku</p>
-        <br />
-        <p>contact: 07044354335</p>
-        <br />
-        <p>email:chideranjoku51@gmail.com</p>
-      </div>
-    </div>
-
-    <footer class="foot">
-      <p>
-        by chidera njoku <br />
-        created:2024
-      </p>
-    </footer>
-  </body>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="contact.css" />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>contact</title>
+  </head>
+  <body>
+    <div>
+      <nav>
+        <ul>
+          <li><a href="homepage.html">Home</a></li>
+          <li><a href="aboutme.html">About Me</a></li>
+          <li><a href="project.html">Projects</a></li>
+          <li style="float: right"><a href="contact.html">Contact</a></li>
+        </ul>
+      </nav>
+    </div>
+    <div class="containor">
+      <div class="heading">
+        <h1>Contact</h1>
+      </div>
+
+      <div class="contact">
+        <p>full name: chidera Njoku</p>
+        <br />
+        <p>github: Vestolord32</p>
```

Comment:
add a link on this text of the github profile. reader can easily navigate the the github profile with it instead of copying the name